src/OmniSharp.DotNetTest/Legacy/LegacyTestManager.cs

```diff
@@ -63,22 +71,42 @@ public override RunDotNetTestResponse RunTest(string methodName, string testFram
                 WorkingDirectory = WorkingDirectory,
                 CreateNoWindow = true,
                 UseShellExecute = false,
-                RedirectStandardOutput = false,
-                RedirectStandardError = false
+                RedirectStandardOutput = true,
+                RedirectStandardError = true
             };
 
             var testProcess = Process.Start(startInfo);
 
-            var results = new List<TestResult>();
+            var output = new StringBuilder();
+            var error = new StringBuilder();
+
+            testProcess.OutputDataReceived += (_, e) =>
```

Comment:
It's a line. At least, that's what I've always observed and what the [docs](https://msdn.microsoft.com/en-us/library/system.diagnostics.process.outputdatareceived(v=vs.110).aspx) say:

> The event is enabled during asynchronous read operations on StandardOutput. To start asynchronous read operations, you must redirect the StandardOutput stream of a Process, add your event handler to the OutputDataReceived event, and call BeginOutputReadLine. Thereafter, **the OutputDataReceived event signals each time the process writes a line to the redirected StandardOutput stream**, until the process exits or calls CancelOutputRead.